### ISSUE TITLE
HealthGAN for Time Series Data

### ISSUE THREAD

**Issue by birdx0810** [yknot/ESANN2019#1]
Hi, I've stumbled across the [paper](https://hal.inria.fr/hal-03158549/document) where HealthGAN is used for generating time-series data for the MIMIC-III Time Series Benchmark dataset. I'm trying to implement HealthGAN using PyTorch for said data and would like to inquire the **dimensions regarding the input and hidden**, i.e. are they (Batch, Seq_Len, Dim) or (Batch, ?). Another question is the **hyperparameters** for reproducing the results for the model, e.g. number of layers, hidden dimension etc. Thanks.
